icu4j/maven-build/maven-icu4j-charset/pom.xml

```diff
@@ -0,0 +1,283 @@
+<?xml version="1.0"?>
+<!--
+* © 2016 and later: Unicode, Inc. and others.
+* License & terms of use: http://www.unicode.org/copyright.html
+*******************************************************************************
+* Copyright (C) 2010-2016, International Business Machines Corporation and    *
+* others. All Rights Reserved.                                                *
+*******************************************************************************
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  
+  <parent>
+    <groupId>com.ibm.icu</groupId>
+    <artifactId>icu4j-root</artifactId>
+    <version>73.1-SNAPSHOT</version>
+    <!-- default relativePath for parent -->
+  </parent>
+  
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.ibm.icu</groupId>
+  <artifactId>icu4j-charset</artifactId>
+  <version>73.1-SNAPSHOT</version>
+
+  <packaging>jar</packaging>
+
+  <name> ICU4J Charset Provider</name>
+  <description>icu4j-charset is a supplemental library for icu4j, implementing Java Charset SPI.</description>
```

Comment:
dumb question... is this just an SPI wrapper, or is this the whole ICU4J charset conversion code + data?

Reply:
I think it's just the SPI wrapper.

Reply:
>I think it's just the SPI wrapper.

Not true. Charset conversion codes and data are in this component.

Reply:
The current comment matches the one currently in maven:
https://repo1.maven.org/maven2/com/ibm/icu/icu4j-charset/72.1/icu4j-charset-72.1.pom

So +1 from me to keep as is. We can mess with it later if it does not reflect reality.